types/docker-file-parser/index.d.ts

```diff
@@ -0,0 +1,21 @@
+// Type definitions for docker-file-parser 1.0
+// Project: https://github.com/joyent/docker-file-parse
```

Comment:
The prokect link is broken.

Reply:
I can update that Friday. I'm out tomorrow. Thanks for pointing this out :100: